app/Actions/Place/GetPlaceCollectionByFilters/GetPlaceCollectionByFiltersAction.php

```diff
@@ -28,24 +31,27 @@ public function execute(GetPlaceCollectionByFiltersRequest $request): GetPlaceCo
     {
         $categoryId = $request->getCategoryId();
         $location = $request->getLocation();
-        $page = $request->getPage() ? : GetPlaceCollectionByFiltersRequest::DEFAULT_PAGE;
+        $page = $request->getPage() ?: GetPlaceCollectionByFiltersRequest::DEFAULT_PAGE;
+        $criterias = [];
 
-        if ($location) {
+        if (!is_null($location)) {
             try {
-                $location = Location::fromString($location);
+                $criterias[] = new GetPlaceByLocationCriteria(Location::fromString($location));
```

Comment:
better to catch Location::fromString($location) InvalidArgumentException separate above, because if GetPlaceByLocationCriteria throws the same exception, we won't know what exactly what cause `InvalidArgumentException`